cmd/server/main.go

```diff
@@ -15,32 +19,27 @@ import (
 	"go.uber.org/zap"
 )
 
+var log *zap.Logger
+var db *sql.DB
+var repo *repository.URLRepository
+
 func main() {
 	// TODO move to separate func
 	// Initialize Logger
-	logger := logger.NewLogger(os.Getenv("LOG_LEVEL"))
-
 	// TODO move to separate func
 	// Connect to Database
-	db, err := handler.InitDB(os.Getenv("DATABASE_URL"))
-	if err != nil {
-		logger.Fatal("Failed to connect to database", zap.Error(err))
-	}
-	defer db.Close()
-
 	// Create Router
 	r := chi.NewRouter()
 
 	// Apply Middlewares
-	r.Use(middleware.Recoverer(logger))
-	r.Use(middleware.RequestLogger(logger))
+	r.Use(middleware.Recoverer(log))
+	r.Use(middleware.RequestLogger(log))
 	r.Use(middleware.CORS)
 	r.Use(middleware.RateLimiter)
 
 	// TODO repository and service must init here together with urlHandler
 	// Initialize Handlers
-	urlHandler := handler.NewURLHandler(db, logger)
-
+	urlHandler := handler.NewURLHandler(db, log)
```

Comment:
fix handler.NewURLHandler it's create new repo and storage but we created it in main